src/index.js

```diff
@@ -1,8 +1,49 @@
 import './style.css';
-import addUser from './modules/adduser.js';
+import createNewGame from './modules/newgame.js';
+import onloadScore from './modules/onloadscore.js';
+import refreshScores from './modules/get.js';
 
 const submitBtn = document.getElementById('submitBtn');
+const refreshBtn = document.getElementById('refreshBtn');
+const game = 'game';
+
+async function getGameId() {
+  if (localStorage.getItem('gameId')) {
+    return localStorage.getItem('gameId');
+  } return createNewGame(game);
+}
```

Comment:
- [ ] I kindly suggest you make use of an `arrow function` rather than what you currently have. It is important to make use of `ES 6` best practices such as using arrow functions instead of the function keyword 👍 ✨